tests/ui/specs/Alert/Alert-specs.js

```diff
@@ -0,0 +1,86 @@
+const Page = require('./AlertPage');
+const {expectClosed, expectOpen, validateTitle} = require('./Alert-utils.js');
+describe('Alert', () => {
+
+	const {alertCommon, components} = Page;
+	beforeEach(() => {
+		Page.open();
+	});
+
+	// Using 5-way
+	describe('using 5-way', () => {
+		it('should spot the fullscreen alert button', () => {
+			expect(alertCommon.buttonFullscreen.isFocused()).to.be.true();
+		});
+
+		it('should spot the cancel button', () => {
+			Page.spotlightSelect();
+			Page.spotlightRight();
+
+			expect(components.alertFullscreen.buttonCancel.isFocused()).to.be.true();
+		});
+		it('should close the fullscreen alert using the ok button', () => {
+			Page.spotlightSelect();
+			Page.spotlightSelect();
+
+			expect(alertCommon.buttonFullscreen.isFocused()).to.be.true();
+		});
+		it('should spot the cancel button', () => {
+			Page.spotlightRight();
+			Page.spotlightSelect();
+			Page.spotlightDown();
+
+			expect(components.alertOverlay.buttonCancel.isFocused()).to.be.true();
+		});
+		it('should close the overlay alert using the close button', () => {
+			Page.spotlightRight();
+			Page.spotlightSelect();
+			Page.spotlightDown();
+			Page.spotlightSelect();
+
+			expect(alertCommon.buttonOverlay.isFocused()).to.be.true();
+		});
+	});
```

Comment:
Maybe you could also add a test for when pressing Esc key (Page.backKey();) and one when clicking outside the alert popup. You can take https://github.com/enactjs/sandstone/pull/945/files as an example.